src/transformers/modeling_bert.py

```diff
@@ -484,7 +537,7 @@ def custom_forward(*inputs):
                     create_custom_forward(layer_module),
                     hidden_states,
                     attention_mask,
-                    layer_head_mask,
+                    head_mask[i],
```

Comment:
This probably isn't strictly safe (head_mask could be None), and only passes because this portion is commented out. I think that's why I created 'layer_head_mask'